modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -0,0 +1,252 @@
+
+.Create a database with Cypher 25 as the default language
+[source, cypher]
+----
+CREATE DATABASE my_new_database DEFAULT LANGUAGE CYPHER 25
+----
+
+[[alter-default-cypher-version]]
+== Alter the default Cypher version for a database
+
+To alter the default Cypher version on a database, add `SET DEFAULT LANGUAGE <language version>` to the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/alter-databases/[`ALTER DATABASE`] command.
+
+[NOTE]
+Selecting a default Cypher version on a database requires the `SET DATABASE DEFAULT LANGUAGE` privilege.
+
+.Set Cypher 5 as the default language on an existing database
+[source, cypher]
+----
+ALTER DATABASE my_new_database SET DEFAULT LANGUAGE CYPHER 5
+----
+
+.Set Cypher 25 as the default language on an existing database
+[source, cypher]
+----
+ALTER DATABASE my_database SET DEFAULT LANGUAGE CYPHER 25
+----
+
+[[show-cypher-versions]]
+== Show the Cypher version of databases
+
+The Cypher version of a database can be seen using the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/listing-databases/[`SHOW DATABASES`] command as follows:
+
+.Retrieve the default language on existing databases
+[source, cypher]
+----
+SHOW DATABASES YIELD name, defaultLanguage
+----
+
+[NOTE]
+`defaultLanguage` is not returned by default.
+It can only be returned with `YIELD`.
+For more information, see the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/standard-databases/listing-databases/#_show_databases_output[Operations Manual -> `SHOW DATABASES` output].
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| name | defaultLanguage 
+
+| "my_database" | "CYPHER 25"
+| "my_new_database"  | "CYPHER 5"
+| "system"           | "CYPHER 25"
+
+2+d|Rows: 3
+|===
+
+[[config-database-upgrades]]
+== Cypher versions, configuration settings, and DBMS upgrades
+
+Changing the Cypher version of new databases can also be done with the setting link:{neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_db.query.default_language[`config_db.query.default_language`].
+This setting determines the default language for new databases where it has not been specified as part of a `CREATE` or `ALTER` database command.
+For example, `config_db.query.default_language=CYPHER_25` will set Cypher 25 as the default language for a DBMS.
+For more information about using configuration settings, see the link:{neo4j-docs-base-uri}/operations-manual/current/configuration/[Operations Manual -> Configuration].
+
+The table below outlines which Cypher version is assigned to databases in different upgrade or installation scenarios:
+
+[cols="3", options="header"]
+|===
+| Scenario | `db.query.default_language` | Databases
+
+| *Standard DBMS upgrade to Neo4j 2025.03+*
+| Unset
```

Comment:
should we in the text above mention what the settings default is or is it enough that ```link:{neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_db.query.default_language[`config_db.query.default_language`]``` does (as I assume that one will say what the default value is)